src/metabase/lib/metadata/calculation.cljc

```diff
@@ -277,7 +277,7 @@
 
 (mr/register! ::display-info
               [:map
-               [:display-name :string]
+               [:display-name {:optional true} :string]
```

Comment:
During the implementation I noticed that `display-info` fails in the BE for most of the drills because they don't provide any `:display-name` property.